packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts

```diff
@@ -1,19 +1,62 @@
-import { fromAssetId } from '@shapeshiftoss/caip'
-import { Asset } from '@shapeshiftoss/types'
+import { AssetId, fromAssetId } from '@shapeshiftoss/caip'
+import { Asset, KnownChainIds } from '@shapeshiftoss/types'
 import { AxiosResponse } from 'axios'
 
 import { SwapError, SwapErrorTypes } from '../../../../api'
 import { bn, bnOrZero } from '../../../utils/bignumber'
 import { ZrxPriceResponse } from '../../types'
-import { zrxService } from '../zrxService'
+import { zrxServiceFactory } from '../zrxService'
+
+export const baseUrlFromChainId = (chainId: string): string | undefined => {
```

Comment:
might make our lives easier if these helper functions throws the `SwapError` `UNSUPPORTED_CHAIN` error so we don't have to check for undefined and throw everywhere they are used

Reply:
Much nicer, good suggestion. Actioned in https://github.com/shapeshift/lib/pull/850/commits/ab4815f5a85a282a06ef92095ac4106b527bbc91.